### ISSUE TITLE
A11y_Azure Tools for VS Code_Azure Function_Filter Subscriptions_Keyboard: Tooltip of 'Filter Subscription' button is not accessible for keyboard only users.

### ISSUE THREAD

**Issue by sohelmulani1998** [microsoft/vscode-azurefunctions#3120]
**GitHubTags:**

#A11yMAS; #A11yTCS; #A11ySev3; #DesktopApp; #Azure Tools for VS Code; #BM_Azure Tools for VS Code_Win32_Apr22; #Win32; #FTP;  #Keyboard; #Linked:Bug153070; #External:VSCode; #WCAG2.1.1; #CEACCReviewed; #A11YCAIOH ;

**Environment Details:**
Application Name: Visual Studio Code Services
Windows Version: Windows 11

**Repro Steps:**

1. Open Visual studio code 
2. Tab till extension button and search for "Azure Functions"
3. Tab till install and and press enter now tab till the "Azure" button and press enter.
4. Tab till filter button and press enter
5. Check if tooltip is accessible to keyboard only users

**Actual:**
Tooltip of 'Filter Subscription' button is not accessible for keyboard only users.

Note: Similar issue observed for the all controls of toolbar as well

**Expected Result:**
Tooltip should be accessible to keyboard only users as well

**Attachment:**
![3120_A11y_Azure Tools for VS Code_Azure Function_Filter Subscriptions_Keyboard](https://user-images.githubusercontent.com/85861804/164215830-4d835cd4-d9ab-408c-bf00-cc6a6bc6ae56.gif)

**Comment by nturinski** [comment]
I've attached a .gif that demonstrates how one would access the filter subscription button with only a keyboard.

![filtertab](https://user-images.githubusercontent.com/5290572/164563565-019addd2-033b-46de-b01f-f86b8e9216bd.gif)

**Comment by sohelmulani1998** [comment]
@nturinski The bus is not about the accessing 'Filter Subscription' button with keyboard. Its about the tooltip associated with it is not accessible to the keyboard only users. and this bug is not just specific to the 'Filter Subscription' button but also for other controls of toolbar.

**Comment by MicroFish91** [comment]
Hi @sohelmulani1998, just to clarify, is the issue that your screen reader is not reading the button tooltip when you select it with your keyboard?  If so, could you please let me know which screen reader you are currently using?  Thank you

**Comment by sohelmulani1998** [comment]
@MicroFish91 This is not a Screen reader issue. This is a Keyboard issue where the tooltips associated with the controls are not accessible to the keyboard only users.
If you observe the gif that I added, you can see on hovering with mouse the tooltips are visible but when navigate with keyboard those tooltips are not visible.

**Comment by MicroFish91** [comment]
@sohelmulani1998 Okay, I think I understand now.  

Please refer to this issue: https://github.com/Microsoft/vscode/issues/52016.  

This is a very special case that only affects users who are using keyboard and are not using a screen reader.  This comment sums up this case quite well: https://github.com/Microsoft/vscode/issues/52016#issuecomment-641096443

VS Code team has not received any user requests to add this feature and it is not supported by modern web browsers, so have no plans to pursue this at the current time.  Thank you for helping us improve our product :)

**Comment by jinglouMSFT** [comment]
@sohelmulani1998 The upstream bug has been closed as by design https://github.com/microsoft/vscode/issues/52016.  How would you like to resolve our bug (this bug)?

**Comment by sohelmulani1998** [comment]
@jinglouMSFT I have opened a new external bug on VS Code as per suggestion from Tanaz. 

New external tracking id: https://github.com/microsoft/vscode/issues/153070

Please track this new bug as external bug.

Thanks!

**Comment by MicroFish91** [comment]
Tracking against: https://github.com/microsoft/vscode/issues/132344

Sev3 justification: https://github.com/microsoft/vscode/issues/132344#issuecomment-946029198

**Comment by MicroFish91** [comment]
I believe the upstream issue has been resolved, can you please confirm and close if so?  Thank you!

**Comment by keerthiduvvuri** [comment]
GithubTags:#ResolvedByTT

**Comment by nturinski** [comment]
I believe that this tag: "GithubTags:#ResolvedByTT" indicates that it should be closed by some sort of bot, but it hasn't been closing any of these issues. I will be that bot.
